lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -2586,6 +2587,9 @@ static void foldChannelShuffle(Function *F) {
 }
 
 void glow::fold(Function *F, CompilationContext &cctx) {
+  // Log the start of current log scope.
```

Comment:
RAII would be a good pattern here.

Reply:
He is using RAII-based macros, in this case we could just remove the LOG_SCOPE_END() define.